live-examples/js-examples/typedarray-buffer.html

```diff
@@ -0,0 +1,9 @@
+<pre>
+<code id="static-js" class="language-js">// create an ArrayBuffer with a size in bytes
```

Comment:
@maboa while you are still working on this one, feel free to remove the `class="language-js"` from these examples. I have cleaned up the rest. Thanks!